bridgeservice/docs/docs.go

```diff
@@ -125,7 +125,7 @@ const docTemplate = `{
         },
         "/claim-proof": {
             "get": {
-                "description": "Returns the Merkle proofs (local and rollup exit root) and",
+                "description": "Returns the Merkle proofs (local and rollup exit root) and\nthe corresponding L1 info tree leaf needed to verify a claim.",
```

Comment:
is there some automated way of multiline instead of putting `/n`?

Reply:
Nope this should be the way, this is what swaggo generated. FWIW, it is rendered properly now. Seems that the common approach is to have a one liner description, but then linter complains about line being too long...

![image](https://github.com/user-attachments/assets/d2250585-f9cd-44c5-9595-d23710db0a68)